Resources/public/css/theme/views/universaldiscovery.css

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.is-universaldiscovery-hidden .ez-universaldiscovery-container {
+    display: block;
+
+    background: rgba(0, 0, 0, 0);
+    -webkit-transform: translateY(-100%);
```

Comment:
Wouldn't it be better to use `transform: translate3d(0, -100%, 0);` to use GPU for animations?